packages/ui/src/components/sha-badge.tsx

```diff
@@ -1,18 +1,20 @@
-import { Text } from '@harnessio/canary'
+import { ReactNode } from 'react'
+
+import { Text } from '@/components'
 
 interface RootProps {
-  children: React.ReactNode
+  children: ReactNode
   className?: string
 }
```

Comment:
No need to change in this PR, but `PropsWithChildren` is a great way to get the "official" way of saying a component accepts children. You can use it in a fews ways...

```typescript
interface RootProps extends PropsWithChildren<unknown> {
  className?: string
}
```

Or as a type
```typescript
type RootProps = PropsWithChildren<{
  className?: string
}>
```